ports/otl/portfile.cmake

```
@@ -1,9 +1,9 @@
-set(OTL_VERSION 40463)
+set(OTL_VERSION 40476)
 
 vcpkg_download_distfile(ARCHIVE
     URLS "http://otl.sourceforge.net/otlv4_${OTL_VERSION}.zip"
     FILENAME "otlv4_${OTL_VERSION}-9485a0fe15a7-1.zip"
-    SHA512 9485a0fe15a737d55b0746a7e289b1a20e9435ed5c69bda7010705f8cde0a456163d83221d0103236a723837596613b578edc6d3d0007ce80a6cc76b4ed83888
+    SHA512 ef43695b4c97d953bd997ba2381ae37ae067efb99a14479e93f502e3ad5d651829a9712bd14698c8663f733114861f6e2ae2277ed0967fad1ac2a068825a9e38
 )
 
 vcpkg_extract_source_archive_ex(
```

```suggestion
vcpkg_extract_source_archive(
```
https://github.com/microsoft/vcpkg/blob/master/docs/maintainers/vcpkg_extract_source_archive.md